src/main/groovy/de/dkfz/config/ResourceSet.groovy

```diff
@@ -0,0 +1,104 @@
+package de.dkfz.config
```

Comment:
Might clash with my latest changes to RoddyCore... :( I also moved the ResourceSet out of the class.